_includes/head.html

```diff
@@ -1,7 +1,11 @@
 <head>
   <meta charset='utf-8' />
   <meta http-equiv="X-UA-Compatible" content="chrome=1" />
-  <meta name="description" content="Up For Grabs : Jump in!" />
+  <meta property="og:url"         content="http://up-for-grabs.net" />
+  <meta property="og:type"        content="product" />
+  <meta property="og:title"       content="Up For Grabs" />
+  <meta property="og:description" content="Want to contribute to open source, but github feels like an ocean? Grab these tiny issues before they dissolve themselves!" />
```

Comment:
I'm gonna defer to @dahlbyk on the words here, but please make it `GitHub` at least here